mne/io/kit/tests/test_kit.py

```diff
@@ -50,6 +51,7 @@ def test_data():
                               hsp=hsp_path, stim=list(range(167, 159, -1)),
                               slope='+', stimthresh=1)
     assert_true('RawKIT' in repr(raw_py))
+    assert_true(raw_mrk.info['kit_system_id'] == 34)
```

Comment:
These are the known ones: https://github.com/mne-tools/mne-python/blob/master/mne/io/kit/constants.py#L110

I was actually wondering whether they should be added to FIFF now, i.e. like

`FIFF.FIFFV_KIT_SYSTEM_UMD_2014 = 53`

Because we're going to need them within MNE to decide for example the layout, or should they stay in the KIT namespace?